docs/snippets/angular/badge-story-starter-example.mdx.mdx

```diff
@@ -1,4 +1,4 @@
-```md
+```mdx
 <!-- Badge.stories.mdx -->
```

Comment:
Not related to this PR, @shilman this HTML-comment style doesn't work with MDX2 anymore, does it?

Reply:
@ndelangen, you're absolutely right about that. It actually doesn't. If a user grabs the code snippet here and pastes it in an MDX 2 example, he's treated with the following:
![Storybook-comment-error-mdx2](https://user-images.githubusercontent.com/22988955/202289553-481335d4-ecf3-4c5f-8f3e-3c303b8b3797.png)

Thanks for the callout nonetheless, really appreciate it 🙏 ! As once the snippets and overall docs are going to be updated for MDX 2, I'll need to keep this in mind and adjust the comment to avoid issues.